db/column_family.cc

```diff
@@ -1272,7 +1263,7 @@ void ColumnFamilyData::InstallSuperVersion(
 
 void ColumnFamilyData::InstallSuperVersion(
     SuperVersionContext* sv_context, InstrumentedMutex* db_mutex,
-    const MutableCFOptions& mutable_cf_options, bool noImmMemoryContribution) {
```

Comment:
Remove changes made during #8454 when the mempurge process was happening in the middle of the SwitchMemtable process.